src/components/NestedAccordionMenu/MenuItems/index.js

```diff
@@ -32,10 +33,18 @@ function MenuItem(props) {
     return found;
   }
 
+  function hasIcons(subItems) {
+    subItems.forEach(itm => {
+      if (itm.Icon) {
+        hasIcon = true;
+      }
+    });
+    return hasIcon;
+  }
+
```

Comment:
I’d extract this to a new file in https://github.com/BLSQ/manager-ui/tree/master/src/utils as you’re also using that in https://github.com/BLSQ/manager-ui/pull/9/files#diff-03c0bafcbd11ad5ac0301f2b346cb7edR16

Reply:
If this is about knowing if at least one subItems has an icon (and updating hasIcon) accordingly maybe use some: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some ?

hasIcon = subItems.some((itm) => itm.Icon)
return hasIcon

Reply:
(and at this time, evaluate if a function is still useful as it's a one liner)

Reply:
@vanakenm thank you, I have changed and now there is no need to use a function anymore